### PR TITLE
VZ-7841: back port document support for OS/OSD plugin installation to release 1.4

### PR DESCRIPTION
**Pull request by karan56625**
document support for OS/OSD plugin installation in release 1.4

### CODE REVIEW

File: content/en/docs/customize/opensearch.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ There are three ways to define a plug-in in the `plugins.installList`:
  - To be compatible, major, minor, and patch plug-in versions must match OpenSearch major, minor, and patch versions. For example, plug-ins versions 2.3.0.x are compatible only with OpenSearch version 2.3.0.
 {{< /alert >}}
 
-For OpenSearch Dashboard, you can provide the plug-ins by defining the field [spec.components.opensearch-dashboards.plugins](/docs/reference/api/vpo-verrazzano-v1beta1/#install.verrazzano.io/v1beta1.v1beta1.OpenSearchDashboardsComponent) in the Verrazzano custom resource.
+For OpenSearch Dashboard, you can provide the plug-ins by defining the field [spec.components.opensearch-dashboards.plugins](/docs/reference/api/verrazzano/v1beta1/#opensearch-dashboards-component) in the Verrazzano custom resource.
```
Comment (rosemarymarano) [inline content/en/docs/customize/opensearch.md:193]: -> the field [spec.components.opensearch-dashboards.plugins](http://localhost:1313/docs/reference/api/vpo-verrazzano-v1beta1/#install.verrazzano.io/v1beta1.v1beta1.OpenSearchDashboardsComponent) 
I'm not seeing this x-ref link going to where you want it to go

File: content/en/docs/reference/API/Verrazzano/v1alpha1.md [hunk 2]
```diff
@@ -294,6 +295,19 @@ spec:
 | `minIndexAge` | [Time]({{<opensearch_docs_url>}}/opensearch/units/) | Amount of time until a managed index is deleted. Default is seven days (`7d`). | No |
 | `rollover` | [Rollover](#opensearch-index-management-rollover) | Index rollover settings. | No |
 
+#### OpenSearch Plugins
+| Field | Type | Description | Required |
+| --- | --- | --- | --- |
+| `enabled` | bool | To enable or disable the OpenSearch plug-ins | Yes |
```
Comment (rosemarymarano) [inline content/en/docs/reference/API/Verrazzano/v1alpha1.md:301]: Each description sentence should end with a period.

**Review by rosemarymarano — APPROVED** [review]

File: content/en/docs/reference/API/Verrazzano/v1alpha1.md (hunk 2 above)
Comment (karan56625) [reply content/en/docs/reference/API/Verrazzano/v1alpha1.md]: fixed.

File: content/en/docs/customize/opensearch.md (hunk 1 above)
Comment (rosemarymarano) [reply content/en/docs/customize/opensearch.md:193]: Might be my mistake, I think you're correct.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**